.github/workflows/pr-actions.yml

```diff
@@ -37,26 +37,34 @@ jobs:
           exit 1
       - run: echo "PR is not Draft"
 
+  # Require a team label
+  version-label:
+    uses: ./.github/workflows/run-npm-script.yml
+    needs: draft
+    with:
+      label: "Check PR Version Label"
+      script: "check-version-label"
```

Comment:
If this fails, none of the other tests will run which is a nice benefit imo.

Reply:
Thinking further about this for our ideal flow. Adding a label doesn't trigger the PR, right? So every PR that gets created will initially fail this check, not run the tests, and then after the label is added, we'll need to manually re-run the tests.

I think it might be better if this check were required by the branch protection policy, but if it ran both on 'normal' pr events like the rest and then also on the 'label' event where we can check if the PR has a version label like in this [SO](https://stackoverflow.com/questions/62325286/run-github-actions-when-pull-requests-have-a-specific-label).

That way the tests can still run when the PR is opened, we can apply the label without needing to re-run the whole test suite, and then it'll check for the version label so that can pass

Reply:
Want to take a crack at that since you have the permissions? I'm fine letting this simmer, it's not a hard requirement at the moment.